.github/workflows/ci.yml

```diff
@@ -61,13 +60,12 @@ jobs:
       - name: Download dependencies
         id: depdownload
         run: |
-          sudo apt-get update
-          sudo apt-get install -y libdevmapper-dev libbtrfs-dev libgpgme-dev
+          hack/build/ci/install_cgo_dependencies.sh
+          hack/build/ci/install_kubebuilder.sh
       - name: Run Unit tests
         id: unittest
         run: |
-          chmod +x ./hack/build/test.sh
-          ./hack/build/test.sh
+          go test -cover -tags integration,containers_image_storage_stub ./...
```

Comment:
this could also be moved into a separate file.
having no more bash logic in the action manifests is the goal here!

Reply:
I'm not really a friend of creating a bash script for one line, but created it for now